mbedtls-sys/build/config.rs

```diff
@@ -363,18 +376,20 @@ pub const DEFAULT_DEFINES: &'static [CDefine] = &[
     ("MBEDTLS_SSL_CID_IN_LEN_MAX",                        Undefined), // default: 32
     ("MBEDTLS_SSL_CID_OUT_LEN_MAX",                       Undefined), // default: 32
     ("MBEDTLS_SSL_CID_PADDING_GRANULARITY",               Undefined), // default: 16
+    ("MBEDTLS_SSL_TLS1_3_PADDING_GRANULARITY",            Undefined), // default: 1
     ("MBEDTLS_SSL_OUT_CONTENT_LEN",                       Undefined), // default: 16384
     ("MBEDTLS_SSL_DTLS_MAX_BUFFERING",                    Undefined), // default: 32768
     ("MBEDTLS_SSL_DEFAULT_TICKET_LIFETIME",               Undefined), // default: 86400
     ("MBEDTLS_PSK_MAX_LEN",                               Undefined), // default: 32
     ("MBEDTLS_SSL_COOKIE_TIMEOUT",                        Undefined), // default: 60
-    ("MBEDTLS_SSL_CIPHERSUITES",                          Undefined), // no default
+    ("MBEDTLS_SSL_CIPHERSUITES",                          Undefined), // default: MBEDTLS_TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
```

Comment:
Actually there's no default. The default is not in config.h. The default is what another header defines the macro as when it's not defined.